frontend/src/time_utils.js

```diff
@@ -8,8 +8,8 @@ export function convertTimestampToDate(timestampDate) {
     .toDateString().slice(4);
 }
 
-/** Format JavaScript date in the form MM-DD-YYYY. */
-export function formatDate(date) {
+/** Format JavaScript date in the form MM/DD/YYYY if mFirst is true, otherwise YYYY/MM/DD. */
+export function formatDate(date, mFirst) {
   var d = new Date(date),
```

Comment:
nit: better variable name than just `d`?

Reply:
to be very honest I cannot think of a better variable name other than date, and it cannot be named date
do you have suggestions?

Reply:
maybe `dateObject`?
i just think single letter variable names might not be the best names lol